### DISCUSSION TITLE
How to keep "/" in column name?

### DISCUSSION THREAD

**Discussion by HuifangYeo** [atoti/atoti#589] [Q&A] [answered]
Hi, 
I have a CSV file with one of the columns named "(P/Km²)". I get the below error:

```
Py4JJavaError: An error occurred while calling o19.createStore.
: java.lang.IllegalArgumentException: The field Density (P/Km²) contains the forbidden datastore schema separator "/".
```

Is there a way to keep the column name as it is?

**Comment by tibdex** [comment, marked as answer]
Not for now but you can use `read_csv()`'s _columns_ parameter to rename it to something like `(P per Km²)`.
